tools/ci_build/build.py

```diff
@@ -500,10 +500,13 @@ def setup_tensorrt_vars(args):
 
         # Set maximum number of iterations to detect unsupported nodes and partition the models for TensorRT
         os.environ["ORT_TENSORRT_MAX_PARTITION_ITERATIONS"] = "1000"
-        
+
         # Set minimum subgraph node size in graph partitioning for TensorRT
         os.environ["ORT_TENSORRT_MIN_SUBGRAPH_SIZE"] = "1"
-        
+
+        # Set FP16 flag
+        os.environ["ORT_TENSORRT_FP16_ENABLE"] = "1"
```

Comment:
why enable fp16 as default during testing?
seems like we should leave as fp32 and enable a subset of tests for fp16.